skimage/measure/_regionprops.py

```diff
@@ -345,11 +345,20 @@ def __getattr__(self, attr):
                     f'Custom regionprop function\'s number of arguments must '
                     f'be 1 or 2, but {attr} takes {n_args} arguments.'
                 )
+        elif attr in PROPS and attr.lower() == attr:
+            # retrieve deprecated property (excluding ancient CamelCase ones)
```

Comment:
```suggestion
            # retrieve deprecated property (excluding old CamelCase ones)
```
because the tests say 'old' (and it's not 'legacy,' right? since you removed the actual property for any CamelCase name)